### ISSUE TITLE
`Cannot get token` when trying to connect to system

### ISSUE THREAD

**Issue by philippbosch** [thomasgermain/pymultiMATIC#96]
This is probably not an issue with pymultiMATIC but maybe somebody can help me. 

I have a script that runs periodically and requests data from our VR921. This has been running without issues for over a year. A few days ago it stopped working and now I get an error message:

```
pymultimatic.api.error.ApiError: Cannot get token
```

When I inspect the reponse I can see this:

```
HTTP Status 401  Unauthorized
The request has not been applied because it lacks valid authentication credentials for the target resource.
```

The official multiMATIC iOS app runs fine and I can also log in on smart.vaillant.com in the browser. So I guess the credentials are not the problem. I also tried a different account – same problem.

I also tried creating a new user, but when clicking on the link in the email to finish the account creation, it takes me to a website that looks very broken (translation strings instead of readable text, API request with 401 in the dev tools, etc.). So to me it looks like something is broken on Vaillant's side. Is anybody else experiencing this?

**Comment by PiotrHab** [comment]
Same here, I'm (or was) getting data each 5 mins. Issue started mid-day Saturday.
I will try 0.6.11 once I get my user right, I can also "confirm account" webpage broken.
Thanks for info and Thomas for great work :)

**Comment by PiotrHab** [comment]
FYI, the "confirm account" webpage worked fine using Chrome.

**Comment by philippbosch** [comment]
@thomasgermain Oh, I didn't see 0.6.11, I only tried 0.6.10. Yes, I can confirm this is "fixed" in 0.6.11. Thank you so much!

**Comment by Q1984** [comment]
Can you please give me a hint how to query vr921 via pymultiMATIC?
For example heating water temperature, pressure, room temp?
Thanks!
